### PR TITLE
Fixed overriding of querystring in internet explorer

### PR DESCRIPTION
**Pull request by seb0zz**
* Added Webstorm IDE directory to .gitignore
* Added check if querystring exists and then only append a cache buster
* Adjusted and also fixed tests (were not working)
* Added info about how to run tests to Readme.md

### CODE REVIEW

**Comment by seb0zz** [comment]
@johntron would be great if you could check and merge my changes if everything is fine for you :+1:

File: index.js [hunk 1]
```diff
@@ -6,7 +6,13 @@ try {
 }
 
 function with_query_strings (request) {
-  request._query = [Date.now().toString()]
+  var timestamp = Date.now().toString()
+  if (request._query !== undefined && request._query[0]) {
+    request._query[0] += '&buster=' + timestamp
```
Comment (johntron) [inline index.js]: We should omit the `buster=` portion - it's possible a user will want to use `buster` themselves.

**Comment by johntron** [comment]
@seb0zz thanks for your PR! If you [remove the `buster=` portion from the query string](https://github.com/johntron/superagent-no-cache/pull/14#discussion_r46222561), I'll merge.

**Comment by seb0zz** [comment]
Thanks for the update, will refactor it asap :+1:

**Comment by seb0zz** [comment]
Sorry took me a little longer but here is the update

**Pull request merged**